package.json

```diff
@@ -1,161 +1,13 @@
-        "@storybook/addon-actions": "^8.2.9",
-        "@storybook/addon-essentials": "^8.2.9",
-        "@storybook/addon-interactions": "^8.2.9",
-        "@storybook/addon-links": "^8.2.9",
-        "@storybook/blocks": "^8.2.9",
-        "@storybook/react": "^8.2.9",
-        "@storybook/react-vite": "^8.2.9",
-        "@storybook/test": "^8.2.9",
-        "@testing-library/jest-dom": "^5.16.2",
-        "@testing-library/react": "^12.1.4",
-        "@types/jest": "^27.4.1",
-        "@types/node": "20.11.0",
-        "@types/react": "17.0.39",
-        "@types/react-csv": "^1.1.3",
-        "@types/react-dom": "17.0.13",
-        "@types/react-router-dom": "^5.3.3",
-        "@types/react-transition-group": "^4.4.4",
-        "@types/recharts": "^1.8.23",
-        "@types/recompose": "^0.30.10",
-        "@typescript-eslint/eslint-plugin": "8.3.0",
-        "@typescript-eslint/parser": "8.3.0",
-        "env-cmd": "10.1.0",
-        "eslint": "^8.56.0",
-        "eslint-config-airbnb": "^19.0.4",
-        "eslint-config-prettier": "^9.1.0",
-        "eslint-import-resolver-typescript": "^3.6.1",
-        "eslint-plugin-import": "^2.29.1",
-        "eslint-plugin-jsx-a11y": "^6.8.0",
-        "eslint-plugin-prettier": "^5.1.2",
-        "eslint-plugin-react": "^7.33.2",
-        "eslint-plugin-react-hooks": "^4.6.0",
-        "eslint-plugin-storybook": "^0.8.0",
-        "husky": "^7.0.4",
-        "jest-extended": "^2.0.0",
-        "jest-junit": "^13.0.0",
-        "jsdom-worker": "^0.2.1",
-        "lint-staged": "12.5.0",
-        "mock-socket": "^9.2.1",
-        "prettier": "^3.1.1",
-        "react-test-render": "^1.1.2",
-        "sass": "^1.69.7",
-        "storybook": "^8.2.9",
-        "ts-jest": "29.2.5",
-        "ts-node": "10.9.2",
-        "typescript": "5.5.4",
-        "vite-plugin-pwa": "0.20.2",
-        "vite-tsconfig-paths": "5.0.1",
-        "vitest": "2.0.5",
-        "workbox-core": "7.1.0",
-        "workbox-navigation-preload": "7.1.0",
-        "workbox-precaching": "7.1.0",
-        "workbox-routing": "7.1.0",
-        "workbox-strategies": "7.1.0",
-        "workbox-window": "7.1.0",
-        "worker-loader": "3.0.8"
+        "lint": "pnpm run -r lint",
+        "build:prod": "pnpm run -r build",
+        "build:libs": "pnpm filter './packages/**' build"
     },
-    "jest": {
-        "collectCoverageFrom": [
-            "**/*.{ts,tsx,js,jsx}",
-            "!**/?(*.)+(mock).+(ts|js)",
-            "!**/node_modules/**",
-            "!**/coverage/**",
-            "!**/serviceWorker.ts",
-            "!**/index.tsx",
-            "!**/setupProxy.js"
-        ],
-        "setupFilesAfterEnv": [
-            "jest-extended"
-        ],
-        "moduleNameMapper": {
-            "monaco-editor": "<rootDir>/node_modules/react-monaco-editor",
-            "monaco-yaml": "<rootDir>/node_modules/react-monaco-editor"
-        }
-    }
+	"devDependencies": {
+		"husky": "^7.0.4"
```

Comment:
let's add lint-staged as well

Reply:
I am levering lint-staged at individual package level. Added the check for lint-staged in husky rc